dom0/sd-usb-autoattach-add.sls

```diff
@@ -6,15 +6,6 @@
 # USB devices to sd-devices.
 ##
 
-# If sys-usb is disposable, we have already set up sd-fedora-dvm to make our
-# modifications in, so we only want to modify sys-usb if it is a regular AppVM
-
-{% set apply = True %}
-{% if grains['id'] == 'sys-usb' and salt['pillar.get']('qvm:sys-usb:disposable', true) %}
-  {% set apply = False %}
-{% endif %}
-
-{% if apply %}
```

Comment:
the reason we aren't worried about this anymore is it's basically saying "only apply these changes if sys-usb is non disposable."  However, if sys-usb is disposable, the changes don't matter anyway since they are identical to the ones in the template (and are applied in the template).

We can probably remove all of this and drop support for non-disposable sys-usb, which we may not be fully supporting in 4.1 anyway.

Reply:
This decision makes sense to me. Thanks for the clear explanation.

Reply:
I was just re-reading `sd-sys-vms.sls` because when we went over it we (or I) thought it didn't support non-disposable `sys-usb` anymore anyway… but after revisiting I think it still technically does.

It doesn't _need_ to be addressed in this PR, but just wanted to note that this PR would technically alter what the code unofficially supports. To go the whole way, we could remove the `sd-sys-vms.sls` check [whether a VM is disposable or not](https://github.com/freedomofpress/securedrop-workstation/blob/main/dom0/sd-sys-vms.sls#L85-L96) and just assume it is.